src/LiveComponent/assets/src/Backend/RequestBuilder.ts

```diff
@@ -33,10 +33,12 @@ export default class {
             0
         );
 
+        const forcePost = props['@attributes']?.['data-force-post'] == true;
```

Comment:
For this, I think we should pass an argument to the constructor of `RequestBuilder`: `string method`. Then, instead of a `data-force-post`, add a new Stimulus value to the Stimulus controller called `requestMethod` that defaults to `post`. Then you can pass that into `Backend` here https://github.com/symfony/ux/blob/643382b746a3354d441b261588f972414fdd6b2b/src/LiveComponent/assets/src/live_controller.ts#L87 - and into `RequestBuilder` here https://github.com/symfony/ux/blob/2.x/src/LiveComponent/assets/src/Backend/Backend.ts#L29

Reply:
Polling will still use "GET" right ?

Reply:
It would not, no. It would be POST all the time. Is there a reason to make polling alone use GET?

Reply:
I've updated the code according to suggestions by @weaverryan.